hdaccount/derivation_path.go

```diff
@@ -0,0 +1,23 @@
+package hdaccount
+
+import "fmt"
+
+type DerivationPath []uint32
+
+var DefaultRootDerivationPath = DerivationPath{44 + 0x80000000, 60 + 0x80000000, 0 + 0x80000000, 0}
+
+func (d DerivationPath) String() string {
```

Comment:
i vendored it because i couldn't figure out https://github.com/ethereum-optimism/supersim/pull/44#discussion_r1673157367

but now that i saw ur fix replaced with the go-ethereum implementation!